ldap2pg/manager.py

```diff
@@ -48,6 +48,11 @@ def query_ldap(self, base, filter, attributes, scope):
             except UnicodeDecodeError as e:
                 message = "Failed to decode data from %r: %s." % (dn, e,)
                 raise UserError(message)
+
+            logger.debug("Evalutate entry: '%s' dn: '%s' attributes: '%s'", entry, dn, attributes)
```

Comment:
This can make logs very big.

`entry` variable contains the tuple of decoded dn and attribute. Thus the messages logs twice dn and attribute.